src/test/java/software/amazon/encryption/s3/S3AsyncEncryptionClientTest.java

```diff
@@ -810,8 +809,8 @@ public void testAsyncInstructionFileConfig() {
 
         S3Client s3Client = S3EncryptionClient.builder()
                 .instructionFileConfig(InstructionFileConfig.builder()
-                        .disableInstructionFile(false)
                         .instructionFileClient(S3Client.create())
+                        .disableInstructionFile(false)
```

Comment:
Do you need to both provide a client and set disable?

Reply:
Clearly not!
https://github.com/aws/amazon-s3-encryption-client-java/blob/ca4b0af97f5f0ea9ed80036b57906eba0b023209/src/test/java/software/amazon/encryption/s3/S3AsyncEncryptionClientTest.java#L562-L565